runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -1805,6 +1805,33 @@ TR_J9ServerVM::getInvokeExactThunkHelperAddress(TR::Compilation *comp, TR::Symbo
    return helper;
    }
 
+bool 
+TR_J9ServerVM::noMultipleConcreteClasses(List<TR_PersistentClassInfo>* subClasses)
```

Comment:
At this point the TR_J9ServerVM implementation is the same as the TR_J9VM.
We talked about changing this to avoid some messages.

Reply:
Please take a look at the latest commit. I added a check in the for loop within this function: 
- if a class is cached on the server, proceed to further check
- if a class is not cached on the server, skip to the next class in list
- once count reaches 2, return
- if all cached classes are checked and count still haven't reach 2, go into the second for loop and check classes that are not cached on the server